frontend/src/metabase/admin/databases/components/DatabaseEditApp/Sidebar/Sidebar.tsx

```diff
@@ -162,6 +187,14 @@ const DatabaseEditAppSidebar = ({
           </SidebarGroup.List>
         </SidebarGroup>
       </SidebarContent>
+      {hasModelActionsSection && (
+        <ModelActionsSidebarContent>
+          <ModelActionsSection
+            hasActionsEnabled={checkDatabaseActionsEnabled(databaseObject)}
+            onToggleActionsEnabled={handleToggleModelActionsEnabled}
```

Comment:
nit: I think the function name should almost be the same as the prop name. e.g.
props = `onDoSomething`
handler = `handleDoSomething`

These are a bit different. But I think in other places, they're like what I've said.

Reply:
Do you mean you'd expect it to be one of:

`onToggleActionsEnabled` + `handleToggleActionsEnabled`
or
`onToggleModelActionsEnabled` + `handleToggleModelActionsEnabled`

?

Reply:
Ahh sorry for being too vague. But yeah that's what I was wondering. Since that's what we do all the time e.g. https://github.com/metabase/metabase/blob/db423bb5cf9ea6d0331996db40740bfee6ef23b9/frontend/src/metabase/core/components/AccordionList/AccordionList.jsx#L579

Reply:
Good catch! Love the:

![greg-miller-consistency-gif-by-rooster-teeth](https://user-images.githubusercontent.com/17258145/209207923-2a7f1ef3-96d4-461b-8153-e964f4b7f014.gif)